lms/static/sass/xmodule/_headings.scss

```diff
@@ -88,29 +88,33 @@ $headings-base-color:                       $gray-d2;
     letter-spacing: 1px;
 }
 
-.xblock .xblock {
+// let's make these classnames available to the entire LMS!
+
+.hd-1,
```

Comment:
Can you move these classes to the following file as they are no longer scoped to XBlocks:

https://github.com/edx/edx-platform/blob/master/lms/static/sass/base/_base.scss